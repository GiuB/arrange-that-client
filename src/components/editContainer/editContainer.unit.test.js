import React from 'react';
import Enzyme, {render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'

import EditContainer from './editContainer';

Enzyme.configure({adapter: new Adapter()});

it('NavAppBar is consistent', () => {
    // Render a NavAppBar
    const editContainer = render(<EditContainer 
        name='van'
        size={8}
        handleNameChange={null}
        handleSizeChange={null}
        handleEnter={null}
        handleEsc={null}/>);

    expect(toJson(editContainer)).toMatchSnapshot();
});