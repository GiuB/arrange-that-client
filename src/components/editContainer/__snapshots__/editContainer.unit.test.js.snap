// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditContainer renders 1`] = `
<div>
  <div
    class="MuiPaper-root-3 MuiPaper-elevation1-6 MuiPaper-rounded-4 MuiCard-root-2"
  >
    <div
      class="MuiCardHeader-root-30"
    >
      <div
        class="MuiCardHeader-avatar-31"
      >
        <h1
          class="MuiTypography-root-36 MuiTypography-headline-41 MuiTypography-alignCenter-50"
        >
          <div
            class="MuiFormControl-root-62 EditContainer-sizeField-1"
          >
            <div
              class="MuiInputBase-root-79 MuiInput-root-66 MuiInput-underline-70 MuiInputBase-formControl-80 MuiInput-formControl-67"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input-89 MuiInput-input-74 MuiInputBase-inputType-92 MuiInput-inputType-77"
                placeholder="Size"
                type="number"
                value="8"
              />
            </div>
          </div>
        </h1>
      </div>
      <div
        class="MuiCardHeader-content-33"
      >
        <h1
          class="MuiTypography-root-36 MuiTypography-headline-41 MuiTypography-alignCenter-50"
        >
          <div
            class="MuiFormControl-root-62"
          >
            <div
              class="MuiInputBase-root-79 MuiInput-root-66 MuiInput-underline-70 MuiInputBase-formControl-80 MuiInput-formControl-67"
            >
              <input
                aria-invalid="false"
                autofocus=""
                class="MuiInputBase-input-89 MuiInput-input-74"
                placeholder="Name"
                type="text"
                value="van"
              />
            </div>
          </div>
        </h1>
      </div>
    </div>
  </div>
</div>
`;

exports[`NavAppBar is consistent 1`] = `
<div>
  <div
    class="MuiPaper-root-3 MuiPaper-elevation1-6 MuiPaper-rounded-4 MuiCard-root-2"
  >
    <div
      class="MuiCardHeader-root-30"
    >
      <div
        class="MuiCardHeader-avatar-31"
      >
        <h1
          class="MuiTypography-root-36 MuiTypography-headline-41 MuiTypography-alignCenter-50"
        >
          <div
            class="MuiFormControl-root-62 EditContainer-sizeField-1"
          >
            <div
              class="MuiInputBase-root-79 MuiInput-root-66 MuiInput-underline-70 MuiInputBase-formControl-80 MuiInput-formControl-67"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input-89 MuiInput-input-74 MuiInputBase-inputType-92 MuiInput-inputType-77"
                placeholder="Size"
                type="number"
                value="8"
              />
            </div>
          </div>
        </h1>
      </div>
      <div
        class="MuiCardHeader-content-33"
      >
        <h1
          class="MuiTypography-root-36 MuiTypography-headline-41 MuiTypography-alignCenter-50"
        >
          <div
            class="MuiFormControl-root-62"
          >
            <div
              class="MuiInputBase-root-79 MuiInput-root-66 MuiInput-underline-70 MuiInputBase-formControl-80 MuiInput-formControl-67"
            >
              <input
                aria-invalid="false"
                autofocus=""
                class="MuiInputBase-input-89 MuiInput-input-74"
                placeholder="Name"
                type="text"
                value="van"
              />
            </div>
          </div>
        </h1>
      </div>
    </div>
  </div>
</div>
`;
